src/diffusers/loaders.py

```diff
@@ -1406,6 +1414,11 @@ def _modify_text_encoder(
                 )
                 lora_parameters.extend(mlp_module.fc2.lora_linear_layer.parameters())
 
+        if is_network_alphas_populated and len(network_alphas) > 0:
+            raise ValueError(
+                f"The `network_alphas` has to be empty at this point but has the following keys \n\n {', '.join(network_alphas.keys())}"
+            )
```

Comment:
Like: https://github.com/huggingface/diffusers/pull/4371/files#r1278549857

Reply:
Can we show ValueError as warning log and move on instead of raise error?

Reply:
I think a ValueError is great here